index.rst

```diff
@@ -42,20 +42,163 @@
 
 .. sectnum::
 
-.. TODO: Delete the note below before merging new content to the master branch.
+.. _introduction:
+
+Introduction
+============
+
+At the summit it is sometimes desirable for instrument data to be analyzed rapidly after data acquisition in order to get quick feedback of data quality.
+The different use cases for processing data are described in :cite:`DMTN-111` and in this document we focus on data processing directly driven by data acquisition.
+
+Data processing must be triggered in some way with enough information to be able to work out which data to process and which analysis pipeline to use.
+The unique identifier for every observation is the "Observation Identifier" (aka "obsid" aka "image name") that is guaranteed by the observatory systems to be unique for every observation regardless of telescope (main telescope vs AuxTel), instrument, or which controller was involved to take the data.
+This obsid is issued by the camera control system whenever data are acquired.
+Additionally, there is a "grouping" concept that can be used by scripts to indicate that observations taken with the same group identifier are related.[*]_
+The group ID is how standard visits will be observed with both snaps being part of the same group.
+The group ID will be provided to each script by the ScriptQueue and will be uniquely defined for that execution of the script.
+The system will guarantee that no observing script will ever be given the same group ID.
+
+Since DM software always mediates data access via a Data Butler this leads to a requirement that raw data must always be retrievable from a butler with a Data ID solely containing the obsid.
+Additionally the detector name can be specified in order to process a subset of the acquired data, and if a group ID is given instead of an obsid, multiple observations can be processed together.
+
+All data products generated using this system are treated as transient data products that will not be archived and will not be required to include permanent provenance tracking.
+The purpose is to provide early feedback to observing in the form of data quality assessments.
+
+.. [*] In :cite:`LTS-836` the group ID is called a "transaction ID" since it is associated with a single queue transaction.
+
+.. note::
+
+  No LSST rapid analysis pipelines are required to co-add data incrementally.
+  Pipelines should not co-add data as it arrives and should always defer co-addition until the final observation in the group has been taken.
+
+Controlling Data Processing from Observing Scripts
+==================================================
+
+In the initial design of the OCS Controlled Pipeline System (OCPS) [*]_ the pipeline processing will be controlled directly by commands sent from the observing script to a CSC.
+The commands will indicate which data to process and the name of the pipeline to execute but they will not include algorithmic code.
```

Comment:
The name will be the name of a gen3 pipeline, for example ProcessCcd.  Versioning should not happen in the observing scripts. We probably want an OCPS configuration that indicates things like an eups tag (if we want to set up entire eups builds). Maybe it could be configured with additional directories that could be setup by eups before processing? @ktl will there be a facility to have a lsst_distrib installation at the summit that can be updated by observatory staff?